src/common/images/imageGallery.component.tsx

```diff
@@ -0,0 +1,435 @@
+              }}
+            >
+              {data.map((card, index) => {
+                const isUndisplayed = !displayedImages?.some(
+                  (img) => img.id === card.row.original.id
+                );
+
+                return isUndisplayed ? null : (
+                  <Card
+                    component={Grid}
+                    item
+                    container
+                    xs
+                    key={`thumbnail-displayed-${index}`}
+                    minWidth={'350px'}
+                  >
+                    <Grid
+                      display="flex"
+                      justifyContent="flex-start"
+                      alignItems="center"
+                      item
+                      container
+                      xs={12}
+                    >
+                      <Grid item xs={2}>
+                        <MRT_SelectCheckbox
+                          row={card.row as MRT_Row<APIImage>}
+                          table={table}
+                          sx={{
+                            ariaLabel: `${card.row.original.file_name} checkbox`,
+                            margin: 0.5,
+                          }}
+                        />
+                      </Grid>
+                    </Grid>
+
+                    <Grid
+                      display="flex"
+                      justifyContent="center"
+                      alignItems="center"
+                      item
+                      minHeight={`${MAX_HEIGHT_THUMBNAIL}px`}
+                      xs
+                    >
+                      <ThumbnailImage
+                        onClick={() =>
+                          setCurrentLightBoxImage(card.row.original.id)
+                        }
+                        image={card.row.original}
+                      />
+                    </Grid>
+
+                    <Grid
+                      display="flex"
+                      justifyContent="center"
+                      alignItems="center"
+                      item
+                      container
+                      xs={12}
+                    >
+                      <Grid xs={2} item>
+                        <MRT_ToggleRowActionMenuButton
+                          cell={card as MRT_Cell<APIImage>}
+                          row={card.row as MRT_Row<APIImage>}
+                          table={table}
+                          sx={{
+                            ariaLabel: `actions ${card.row.original.file_name} photo button`,
+                            margin: 0.5,
+                          }}
+                        />
+                      </Grid>
+                      <Grid item xs={8}>
+                        <OverflowTip
+                          sx={{
+                            fontVariant: 'body2',
+                            textAlign: 'center',
+                          }}
+                        >
+                          {card.row.original.file_name}
```

Comment:
A user feedback thing, but not sure how useful the filename is at a glance compared to the title or description. A related though though, it could have the title or description as a tooltip on the image? That way you can look at them quicker without opening up dialogue.

Reply:
I don't think that's need. they have access to the data using other means

Reply:
Sure, depends on how they use it as well.